### PR TITLE
Updating operatorhub resources

### PR DESCRIPTION
**Pull request by RakhithaRR**

### CODE REVIEW

File: operatorhub-resources/api-manager-operator/api-manager-operator.package.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 packageName: api-manager-operator
```
Comment (pubudu538) [inline operatorhub-resources/api-manager-operator/api-manager-operator.package.yaml]: This has to change as wso2am-operator... Wherever you see api-manager-operator, change it as wso2am-operator

File: operatorhub-resources/api-manager-operator/api-manager-operator.package.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 channels:
   - name: stable
-    currentCSV: api-manager-operator.v1.0.0
+    currentCSV: api-manager-operator.v1.0.1
```
Comment (pubudu538) [inline operatorhub-resources/api-manager-operator/api-manager-operator.package.yaml]: version should be v1.0.0

File: operatorhub-resources/local-testing/operator-source.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: rakhitharr-operators
+  namespace: marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: rakhitharr
+
```
Comment (pubudu538) [inline operatorhub-resources/local-testing/operator-source.yaml:10]: What is this file? Can we remove ur name from here?

Comment (pubudu538) [reply operatorhub-resources/local-testing/operator-source.yaml:10]: So this is quay.io repo?

**Review by pubudu538 — CHANGES_REQUESTED** [review]

File: operatorhub-resources/local-testing/operator-source.yaml (hunk 3 above)
Comment (RakhithaRR) [reply operatorhub-resources/local-testing/operator-source.yaml:10]: The bundle image is currently in my quay.io repo.

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**